admin/lib/validator/pupil-validator.js

```diff
@@ -153,6 +159,7 @@ module.exports.validate = async (pupilData, schoolId, isMultiplePupilsSubmission
     }
   }
   if (upnErrorArr.length > 0) {
+    // @ts-ignore
```

Comment:
This shouldn't need to be ignored.

Reply:
its an array, being passed to a method that expects a string.  From what i recall, the code appeared to be lazy in the sense that the author probably knew there would only ever be one element in `upnErrorArr`.  `validationError.addError` has no concept of dealing with arrays.

Reply:
The calling code probably needs to be refactored then.  We can use these hints as a ToDo list...